### ISSUE TITLE
On disk_space alarm

### ISSUE THREAD

**Issue by sohwaje** [netdata/netdata#5107]
I want to continue the alarm until the problem is solved.
But now only once. 

> # low disk space

> # checking the latest collected values
># raise an alarm if the disk is low on
># available disk space

>   alarm: disk_space_HOME
>      on: disk_space._home
>      os: linux freebsd
>   hosts: *
>families: /home
>    calc: $used * 100 / ($avail + $used)
>   units: %
>   every: 1m
>    warn: $this > (($status >= $WARNING ) ? (50) : (60))
>    crit: $this > (($status == $CRITICAL) ? (90) : (98))
>   delay: up 1m down 15m multiplier 1.5 max 1h
>   info: current disk space usage
>      to: sysadmin

**Comment by cakrit** [comment]
If I understand correctly, what you are asking for is a duplicate of #2956 
It has high priority, we will get on it soon. Can you verify so we can close this one?

**Comment by sohwaje** [comment]
Thank you for reply.
I understood.
